libs/domains/service-helm/feature/src/lib/networking-port-setting-modal/networking-port-setting-modal.tsx

```diff
@@ -169,6 +231,20 @@ export function NetworkingPortSettingModal({ port, onClose, onSubmit }: Networki
               />
             )}
           />
+          <Controller
+            name="namespace"
+            defaultValue=""
```

Comment:
Is it necessary to set an `defaultValue` empty for each controller?

Reply:
IMO this was useless, I remove them
https://github.com/Qovery/console/pull/1654/commits/dfc81efd64629e7d3d8219efc4c24d41535403b4